golem-worker-service/src/grpcapi/worker.rs

```diff
@@ -116,6 +119,25 @@ impl GrpcWorkerService for WorkerGrpcApi {
         }))
     }
 
+    async fn get_worker_metadatas(
+        &self,
+        request: Request<GetWorkerMetadatasRequest>,
+    ) -> Result<Response<GetWorkerMetadatasResponse>, Status> {
+        let response = match self.get_worker_metadatas(request.into_inner()).await {
+            Ok((cursor, workers)) => {
+                get_worker_metadatas_response::Result::Success(GetWorkerMetadatasSuccessResponse {
+                    workers,
+                    cursor: cursor.unwrap_or(0), // FIXME can we do option in proto?
```

Comment:
I guess we can do option in proto?

Reply:
in proto3, everything is optional, 

just i think tokio-rs/prost, for types like u64 (which have default value in rust), do not making them optional 

but custom type/message 

```proto
message Cursor {
   uint64 value = 1;
}
```
can solve that

Reply:
I think you can write `optional` in proto and prost will handle that. (I was surprised by this once already because I also thought everything is optional in proto3)